doc/word.xml

```diff
@@ -0,0 +1,46 @@
+#############################################################################
+##
+#W  word.xml
+#Y  Copyright (C) 2020                                      Maria Tsalakou
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+
+<#GAPDoc Label="WordToString" >
+<ManSection>
+  <Oper Name = "WordToString" Arg = "alphabet, word"
+    Label="for a string and a list"/>
+  <Returns>
+    The <A>word</A> in the form of a string.
+  </Returns>
+  <Description>
+    Returns the word <A>word</A> that is given in the form of a list, in 
+    the form of a string of letters of the <A>alphabet</A>.
+ <Example><![CDATA[
+ gap> WordToString("abcd", [4, 2, 3, 1, 1, 4, 2, 3]);
```

Comment:
```suggestion
gap> WordToString("abcd", [4, 2, 3, 1, 1, 4, 2, 3]);
```
I've just noticed this since I've been working on a PR building on this one. There's a space here which causes `SEMIGROUPS.TestManualExamples()` to be unable to run properly, and so if `SemigroupsTestAll()` is failing for you, I think this is why.